packages/terra-switch/src/terra-dev-site/doc/example/DefaultSwitch.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import Switch from 'terra-switch';
+
+export default () => {
+  const [isOn, setValue] = useState(false);
```

Comment:
We should align the getter and setter names for state.

Reply:
changed name to value in this [commit](3ccbde10c12629e3a8a6a617536b9c403ec693c7)